### PR TITLE
Add ggplot example and fix README image links

### PR DESCRIPTION
**Pull request by StevenMMortimer**
Add new code for section 1 of chapter 7, in order to show a worked ggplot2 example

Update the README image links so that they hit the raw image content

### CODE REVIEW

**Comment by StevenMMortimer** [comment]
@jgendron We have a merge conflict I believe because of the images being switched around. You might want to just look at my code and incorporate locally, then close the pull request. It's probably easier than resolving the conflicts.

**Comment by jgendron** [comment]
I was changing folder names. I did do merge and then Git pulls. The repos
were all lined up. I'm guessing you did a pull and it was to many changes.

Darn, I thought I had it. Still a rookie.

On Friday, May 6, 2016, Steve Mortimer notifications@github.com wrote:

> @jgendron https://github.com/jgendron We have a merge conflict I
> believe because of the images being switched around. You might want to just
> look at my code and incorporate locally, then close the pull request. It's
> probably easier than resolving the conflicts.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/jgendron/com.packtpub.intro.r.bi/pull/3#issuecomment-217557395

## 

http://www.linkedin.com/in/jaygendron

File: Chapter8-ShinyDashboards/campaign-creator-app/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 A Shiny app designed for a custom cluster count and method, and access to a list of records based on cluster membership.
 
-![app_image](https://github.com/jgendron/com.packtpub.intro.r.bi/blob/master/campaign-creator-app/www/campaign-creator-app-screenshot.png)
+![app_image](https://github.com/jgendron/com.packtpub.intro.r.bi/blob/master/Chapter8-ShinyDashboards/campaign-creator-app-screenshot.png)
```
Comment (jgendron) [inline Chapter8-ShinyDashboards/campaign-creator-app/README.md]: The images seem to be working on the git

File: Chapter8-ShinyDashboards/expenditures-revenue-app/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 A simple Shiny App that predicts revenue from user inputted marketing expenditure via a linear regression model.
 
-![app_image](https://github.com/jgendron/com.packtpub.intro.r.bi/blob/master/Chapter8-ShinyDashboards/expenditures-revenue-app.png)
+![app_image](https://github.com/jgendron/com.packtpub.intro.r.bi/blob/master/Chapter8-ShinyDashboards/expenditures-revenue-app-screenshot.png)
```
Comment (jgendron) [inline Chapter8-ShinyDashboards/expenditures-revenue-app/README.md]: The images seem to be working on the git

**Comment by StevenMMortimer** [comment]
@jgendron I've added a `stopifnot` check in the campaign creator app's `global.R` file that will stop the application from launching if the user's version of the `DT` package is not current enough for the proper rendering of the currency formatting.

Here is the link to the code commit if you'd like to see: https://github.com/jgendron/com.packtpub.intro.r.bi/pull/3/commits/ad8384f4c42750e73c4aae1ee9d359c7f8cbf701

**Comment by jgendron** [comment]
Manually added material. Closing pull request after coordinating with Steve

**Pull request closed without merging**